guides/common/modules/proc_migrating_project_or_smartproxy_using_backup.adoc

```diff
@@ -0,0 +1,34 @@
+[id="migrating-{project-context}-or-proxy-using_backup_{context}"]
+= Migrating {Project} or {SmartProxy} to {EL}{nbsp}9
+
+You can migrate your existing {Project} and {SmartProxy} installation from {EL}{nbsp}8 to a freshly installed {EL}{nbsp}9 system.
+ifdef::satellite[]
+The migration process involves creating a backup of the existing {Project} and {SmartProxy} installation, which is then restored or cloned on the new {EL}{nbsp}9 system.
+Note that cloning only works for {Project} backups and not for {SmartProxy} backups.
+endif::[]
+ifndef::satellite[]
+The migration process involves creating a backup of the existing {Project} and {SmartProxy} installation, which is then restored on the new {EL}{nbsp}9 system.
+endif::[]
+In the below procedure, the "Source server" refers to the existing {EL}{nbsp}8 system on which {Project} or {SmartProxy} is installed, while "Target server" refers to the new {EL}{nbsp}9 system to which {Project} or {SmartProxy} will be migrated.
+
+.Procedure:
+. Ensure that you have enough space for the backup.
+For more information, see {AdministeringDocURL}administering_red_hat_satellite/backing-up-satellite-server-and-capsule_admin#Estimating_the_Size_of_a_Backup_admin[Estimating the size of a backup] in _{AdministeringDocTitle}_.
+. Perform a full backup of the {Project} or {SmartProxy}.
+For more information, see {AdministeringDocURL}backing-up-satellite-server-and-capsule_admin#Performing_a_Full_Backup_admin[Performing a full backup of Satellite Server or Capsule Server] in _{AdministeringDocTitle}_.
+. Deploy a new {EL}{nbsp}9 system.
+Choose the appropriate method to install {Project} or {SmartProxy}:
```

Comment:
Makes sense to me, I was a little iffy about the definitions in the intro, since they can be easily missed and the usage of "in the below procedure" made it sound somewhat clunky too. :)